nav2_bt_navigator/behavior_trees/navigate_w_replanning_only_for_invalid_path.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+  This Behavior Tree replans the global path when it becomes invalid
+-->
+
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <PipelineSequence name="NavigateWithReplanning">
+        <Sequence>
+            <IsPathValid>
```

Comment:
This still wouldn't update except for the rate controller passing. I think in this situation, you should remove the rate controller, no? That way if it `SUCCEED` then we shouldn't plan, but anytime we `FAIL` we do. I think then the parent node here would be a reactive fallback node instead, yes? Since the compute path should only be done if we fail, and pass when we succeed.